scene/gui/graph_edit.h

```diff
@@ -345,6 +345,7 @@ class GraphEdit : public Control {
 	void remove_valid_left_disconnect_type(int p_type);
 
 	void set_scroll_ofs(const Vector2 &p_ofs);
+	void _set_scroll_ofs(const Vector2 &p_ofs);
```

Comment:
If the added method is supposed to be private, it needs to be put into the appropriate section above.